schema/core.md

```diff
@@ -97,19 +93,20 @@ this approach was adopted from ESIP Science on Schema Org.
 A single author can be expressed as:
 
 ``` json
-  ...
+{
   "creator": {
     "@type": "Person",
     "name": "John Doe",
     "email": "john.doe@email.com",
```

Comment:
I'm assuming you removed the "..." so that each of these examples includes valid JSON. If that's the case, we need to pay attention to other things like the trailing comma on the "email" element, which will invalidate this snippet. There are several examples that have this issue.

Reply:
I agree with @horsburgh. If we want to make these valid JSON by removing `...`, then let's also remove all the trailing commas.

Reply:
Done! I validated them using the Schema validator tool.  The commit 394ae2f solved this issue.